R/create_target_design.R

```diff
@@ -207,19 +202,14 @@ create_eval_design <- function(
       } else {
         offset <- 0
         time <- NULL
-        if(type_eval %in% c("cmin", "trough")) {
-          when <- "cmin"
-        } else if (type_eval %in% c("cmax", "peak")) {
-          when <- "cmax"
-        } else if (type_eval %in% c("auc24")) {
-          offset <- 24
-          when <- "dose"
-        } else if (type_eval %in% c("auc12")) {
-          offset <- 12
-          when <- "dose"
-        } else { # cum AUC
-          when <- "dose"
-        }
+        switch(
+          targettype,
```

Comment:
```suggestion
          type_eval,
```

i think this copy/paste error is causing errors in CI

at some point we might want to put this logic in a function since it is identical in two places